src/main/java/com/google/alpollo/model/SongSentiment.java

```diff
@@ -0,0 +1,15 @@
+package com.google.alpollo.model;
+
+/**
+ * Class used to store only sentiment score and magnitude.
+ * It will be easier to send necessary info to front-end as a JSON.
+ */
+public class SongSentiment {
```

Comment:
Why this class doesn't have getters/setters? You can write in documentation that this class is used only for converting its instances to json strings or smth like that.

Reply:
Well, isn't that exactly what it says? Should I rephrase it?

Reply:
Maybe you meant it in "store only sentiment score and magnitude", I didn't get it. Yes, I think it's better to rephrase it.